test/Vault.spec.ts

```diff
@@ -322,6 +325,41 @@ describe('Vault', () => {
     });
   });
 
+  describe('transferOwnership', () => {
+    it('can only be called by the current owner', async () => {
+      await expect(
+        vault.connect(alice).transferOwnership(alice.address),
+      ).to.be.revertedWith('Ownable: caller is not the owner');
+    });
+
+    it('reverts if new owner is address(0)', async () => {
+      await expect(
+        vault.connect(owner).transferOwnership(constants.AddressZero),
+      ).to.be.revertedWith('VaultOwnerCannotBe0Address');
+    });
+
+    it('changes ownership to the new owner', async () => {
```

Comment:
IMHO I don't think this is adding much value to the tests since only the owner can call this method, but if other reviewers agree I can make the change

Reply:
no, there's no point in that expectation, the call would fail if alice was already the owner eitherway